### DISCUSSION TITLE
Hall Effect Sensor new Video & Install instructions

### DISCUSSION THREAD

**Discussion by River-Mochi** [Ralim/IronOS#1647] [Ideas]
### Pinecil only
The original ariticle I wrote in IronOS explained how the Hall effect sensor (HES) interacts with the firmware and the menu screen.
It was not fitting in a software article to show how to mechanically install the HES, but after many requests in Pinecil chat for help on how to physically install the sensor I  wrote an install article.

- Complete instructions, photos, and video here: https://wiki.pine64.org/wiki/Pinecil_Hall_Effect_Sensor
- recommend after installing the sensor to visit IronOS for [menu instructions here](https://ralim.github.io/IronOS/HallSensor/).
<br>

### TL;DR:  

🐧  [Easy trick](https://www.youtube.com/watch?v=aK01V5DrrVk) to open pinecil (no damage :ambulance:)
🎞️   [Install video](https://www.youtube.com/watch?v=vU-fhELpI8Y) made by Ungineering.
<br>

**Comment by River-Mochi** [comment]
### Hints

🎞️ 1. Don't use too many neomydium magnets, It can alter the magnetic field and break the HES affect. 

   - Slowly watch 🎞️ **[this video](https://www.youtube.com/shorts/afkqKwCX00I)**. magnets are already on the metal stand at 12:00 noon, the  HES  works when you see Zzzz (sleeping on detailed mode). Then every time too many magnets are added, it does not help, it breaks the sensor effect. 
   - For this stand, only 2x small magnets were needed for 360 ° coverage (one at 11:50AM and 9:00AM), even better if magnets are taped to the black plastic ring of the stand. Hall senstivity is [ 7 ] in the Sleep menu (need to experiment), Also check the Debug menu Hall number.
  
⛔  2. Avoid hot air install. people damaged their sensor with too much heat. Just use Soldering iron, Magnifying glases to see it better, and lots of flux. Yes, too much heat can make the leg fall out of a chip.

:broom: 3. After install, clean it really well with IPA (isopropyl alcohol) and a soft tooth brush or paint brush  and blow it dry. Leaving it dirty with flux can interfere with i2C and cause noise on the OLED screen.
